pkg/apis/elemental.cattle.io/v1beta1/zz_generated_deepcopy.go

```diff
@@ -22,7 +22,6 @@ limitations under the License.
 package v1beta1
 
 import (
-	config "github.com/rancher/elemental-operator/pkg/config"
```

Comment:
these changes are the result of calling `go generate`

Reply:
This has me worried. I can seem to understand the relationship between your change and why was this auto-generated. I feel dumb

Reply:
There is no relation I believe we just missed to call the code generation on a previous PR. Probably you remember, I think I saw some time ago a repository that had a `validate` target on the make file that was actually checking if the generated code was introducing changes or not. I'd like to have a check in the CI the that tells me I should call `go generate` and commit again as the changes affect the generated code.